download.html

```diff
@@ -10,10 +10,6 @@
   <img src="../images/operating-systems/windows100.png" alt="Windows Icon">
   <p>TripleA&nbsp;for Windows <strong>(64&nbsp;bit)</strong></p>
```

Comment:
Actually I'd remove the (64-bit) part, to treat it as "default" version.

However I can see that some users might assume that the right version gets picked for them automatically and wonder why their version is not really working.

Reply:
My thoughts on keeping it in were to help notify a 32 bit user that it is not right. Hopefully they have been exposed to enough 64 vs 32 that they are watching for it.

For the most part web users do not read a page and they click on the first thing that looks right and click back if they are wrong. In that case they did not read the 64 bit part to begin with and clicked on the windows button.

Reply:
Yeah I assumed you were going for that, but when I learned a thing it's: Never overestimate the users abilities ^^